site/about/deployment/deployment-options.qmd

```diff
@@ -0,0 +1,70 @@
+---
+title: "Deployment options"
+date: last-modified
+listing:
+  - id: validmind-deployment-options
+    type: grid
+    max-description-length: 250
+    sort: false
+    grid-columns: 2
+    fields: [title, description]
+    contents:
+      - multi-tenant-cloud.qmd
+      - virtual-private-validmind.qmd
+---
+
+Choose the {{< var vm.product >}} deployment option that best suits your organization, ensuring secure data protection and seamless integration with your existing environment.
+
+## Available options
+
+Our deployment options provide a balance of control and cost-efficiency while integrating seamlessly with your infrastructure. For added flexibility, you can deploy on Amazon Web Services (AWS), Google Cloud Platform (GCP), or Microsoft Azure.
+
+We offer two deployment models:
+
+- **Multi-tenant cloud** — Multiple organizations (tenants) share infrastructure while keeping data isolated, providing cost-efficiency and scalability. VPC support allows secure network connections.
```

Comment:
Multi-tenant cloud — Multiple organizations (tenants) share infrastructure while keeping data isolated, ensuring cost-efficiency and scalability. For secure connectivity, a private link can be established to ensure traffic stays within your network, avoiding the public internet.

Reply:
Changed verbatim to your suggestion.